src/sgame/sg_main.cpp

```diff
@@ -184,6 +184,8 @@ Cvar::Cvar<int> g_floodMaxDemerits("g_floodMaxDemerits", "client message rate co
 Cvar::Cvar<int> g_floodMinTime("g_floodMinTime", "mute period after flooding, in milliseconds", Cvar::NONE, 2000);
 Cvar::Cvar<int> g_teamStatus("g_teamStatus", "allow /teamstatus command. 0 = disabled, otherwise you can only /teamstatus every <g_teamStatus> seconds.", Cvar::NONE, 5);
 
+Cvar::Cvar<int> g_tacticMilliSeconds("g_tacticMilliSeconds", "clients can only /tactic every <g_tacticSeconds> seconds.", Cvar::NONE, 1000);
```

Comment:
That is a typo. I merely wanted to stick to integers.

Reply:
Sounds good to me, I don't really have a _strong_ opinion about this either way. Granularity isn't a bad thing.

Reply:
I think one second can be too long in some use cases. A user could use multiple commands to set bots to different behaviors. `g_tacticMilliSeconds` is mainly to avoid hitting the flooding protection.